help_to_heat/portal/download_views.py

```diff
@@ -78,6 +105,38 @@ def create_referral_csv(referrals, file_name):
     return response
 
 
+def create_referral_all_xlsx(referrals, file_name):
```

Comment:
in hindsight this is probably better named as something like `create_referral_xlsx_no_pii`